driver/docker/driver.go

```diff
@@ -34,6 +34,9 @@ func (d *Driver) Version(ctx context.Context) (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(driver.ErrNotConnecting, err.Error())
 	}
+	if bkversion, _ := resolveBuildKitVersion(v.Version); bkversion != "" {
+		return bkversion + "-moby", nil
```

Comment:
`-moby` suffix SGTM, WDYT @thaJeztah @tonistiigi?

Reply:
Also let me know for the `+unknown` for future 20.10/23.0 releases that don't support the API yet but we don't have the right tag/commit available yet.

Reply:
I don't care about the suffix in here because we already have a driver property that shows if it is embedded version. But feel free to keep if you think it is important. This is different for `User-Agent` where I would prefer if suffix, that shows it is an embedded version. But that change would not be here but in the moby repo.

Reply:
Removed the suffix.